### ISSUE TITLE
Category name inversed for Android

### ISSUE THREAD

**Issue by TwinProduction** [mozilla/addons#700]
### Describe the problem and steps to reproduce it:
See two following locations: 
- [Android Firefox browser extensions](https://addons.mozilla.org/en-US/android/extensions/)
- [Desktop Firefox browser extensions](https://addons.mozilla.org/en-US/firefox/extensions/)

### What happened?
For Android, there's a category named "Privacy & Security".
For Desktop, there's a category named "Security & Privacy".

### What did you expect to happen?
Both categories are the same, thus they should be named the same and their order should not be inversed.

### Anything else we should know?
I realize this is purely aesthetic, but having the two following urls:
- https://addons.mozilla.org/en-US/android/extensions/security-privacy/
- https://addons.mozilla.org/en-US/firefox/extensions/privacy-security/

does not match the expected pattern:
`https://addons.mozilla.org/en-US/` **{platform}** `/extensions/` **{category}**
because the category is not the same on both platforms.

**Comment by bobsilverberg** [comment]
@jvillalobos can you please confirm whether this is in fact a bug or not?

**Comment by jvillalobos** [comment]
The categories for Firefox and Firefox for Android are distinct in the DB, but in some cases are named the same in the front-end. For example, there's a Shopping category for both. They're different categories, but with the same name and same URL slug. The Security & Privacy example above shows the opposite, where we tried to make them easier to tell apart in the front-end.

There's a trade-off there. We either use the same metadata for consistency (as being requested in this issue), or we use different metadata to make it clearer they're different categories, and also make it easier to tell them apart in analytics.

... and I just remembered there's a pending issue to unify categories (https://github.com/mozilla/addons-server/issues/5989), so I'm just going to close this as something that will be fixed once we do that.
